kubernetes-AKS/mongodb.yaml

```
@@ -0,0 +1,93 @@
+  ports:
+    - port: 27017
+---
+# PVC for MongoDB
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mongodb-pvc
+  namespace: pps-namespace
+spec:
+  # Use managed-csi for Standard SSD managed disks in AKS
+  storageClassName: "managed-csi" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)